Abstract/portable-formula.rb

```diff
@@ -49,6 +58,21 @@ def archs
       return [Hardware::CPU.arch_32_bit]
     end
 
+    # On Tiger and Leopard, override the default behaviour.
+    # Normally we don't build 64-bit there, because the linker is
+    # temperamental, and so "universal" builds don't usually happen.
+    # However, for the purposes of distributing portable packages,
+    # it's very useful to be able to build i386/ppc binaries for use
+    # on both Intel and PowerPC Macs. The Apple-provided compilers
+    # are capable of this on both Intel and Mac hosts.
+    if MacOS.version < :snow_leopard
+      if build.with? "universal"
+        return [:i386, :ppc]
+      else
+        return [Hardware::CPU.arch_32_bit]
```

Comment:
I hardcoded here because we don't have helpers for "give me both 32-bit arches for PPC-era Macs"; it's a niche enough usecase I don't think we need it as a general case.